### ISSUE TITLE
Test Scenario not reported

### ISSUE THREAD

**Issue by sylvain-viole** [Shelex/cypress-allure-plugin#114]
**Describe the bug**
On a specific Gherkin test scenario, whatever the result, the test is not reported in the generated allure report

**To Reproduce**
Hard to reproduce on your side.
100% reproductible on my side

Here is my suite : 

On cypress side :
![image](https://user-images.githubusercontent.com/71819292/147498807-14ea8550-923b-4344-a555-2ef46447eb3c.png)

On allure side :
![image](https://user-images.githubusercontent.com/71819292/147498703-e702ddf6-5969-4278-b8de-e63d41820b17.png)

Scenario #13 = `Sign a signature request` is missing

**Environment (please complete the following information):**
- Cypress version:                               8.7.0
- Cypress Testing library :                  8.0.2
- Cypress allure plugin :                      2.22.0
- OS:                                                     Mac (local) and Linux (CI)

** Developer Tools Console Output **
```
allure-plugin:cy command:end $Command +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:mocha EVENT_TEST_PASS: Sign a signature request Testasync: 0body: "function () {\n      const state = window.testState;\n      return cy\n        .then(() => state.onStartScenario(scenario, indexedSteps))\n        .then(() =>\n          resolveAndRunBeforeHooks.call(this, scenario.tags, state.feature.name)\n        )\n        .then(() =>\n          indexedSteps.forEach((step) =>\n            stepTest.call(this, state, step, rowData)\n          )\n        )\n        .then(() => state.onFinishScenario(scenario));\n    }"callback: ƒ done(err)commands: (157) [{…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, …]ctx: Context {_runnable: undefined, test: undefined, waitRequest: undefined, table: undefined, field_signer: undefined, …}duration: 24246file: nullfinal: truehooks: []id: "r15"invocationDetails: {function: 'Object.runTest', fileUrl: 'https://yousign.app/__cypress/tests?p=cypress/inte…n/features/signature%2520request/createSR.feature', originalFile: 'node_modules/.pnpm/cypress-cucumber-preprocessor@4…cumber-preprocessor/lib/createTestFromScenario.js', relativeFile: 'node_modules/.pnpm/cypress-cucumber-preprocessor@4…cumber-preprocessor/lib/createTestFromScenario.js', absoluteFile: '/Users/sylvain/repos/v3_e2e/node_modules/.pnpm/cyp…cumber-preprocessor/lib/createTestFromScenario.js', …}order: 13parent: Suite {title: 'Create a Signature Request', ctx: Context, suites: Array(0), tests: Array(14), pending: false, …}pending: falseretries: ƒ (...args)state: "passed"sync: truetimedOut: falsetimer: nulltimings: {lifecycle: 32, before each: Array(2), test: {…}, after each: Array(3)}title: "Sign a signature request"type: "test"wallClockDuration: 24326wallClockStartedAt: Mon Dec 27 2021 19:34:52 GMT+0100 (heure normale d’Europe centrale) {}_condensedHooks: true_currentRetry: 0_enableTimeouts: false_events: {error: ƒ}_eventsCount: 1_fired: {runner:test:before:run: true, runner:test:before:run:async: true, runner:runnable:after:run:async: true, runner:test:after:run: true}_retries: 2_slow: 10000_testConfig: {testConfigList: Array(0), unverifiedTestConfig: {…}}_timeout: 12000[[Prototype]]: Runnable +24s
common.js:113 allure-plugin:cy finished: then Object +4ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +1ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +1ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +1ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +1ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: fillOTPCode Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: mailosaurGetMessageById Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: mailosaurGetMessage Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +1ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +1ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +1ms
common.js:113 allure-plugin:cy finished: then Object +0ms
common.js:113 allure-plugin:cy tracked info about command: Object +0ms
common.js:113 allure-plugin:allure updating test AllureTest to have status:passed and details: undefined +24s
common.js:113 allure-plugin:allure set passed for test: Sign a signature request Test +0ms
common.js:113 allure-plugin:mocha EVENT_TEST_END: Sign a signature request Test +18ms
```
**Additional context**

![image](https://user-images.githubusercontent.com/71819292/147499065-26bbe2fe-9e6a-4fdf-957b-a7a819c7d092.png)

![image](https://user-images.githubusercontent.com/71819292/147499229-78fb2b59-db31-4bf2-a05d-413a80d49e1d.png)

- Same behavior with or without Cypress Retry
- Other scenarios are correctly logged and reported

**Comment by sylvain-viole** [comment]
## UPDATE ##

Scenario is not reported at all, whatever the result FAIL or PASS

**Comment by sylvain-viole** [comment]
## UPDATE ##

Made some research, turns out `allure.testParameter` does not throw an error when asked to log a value that is an object

**Comment by Shelex** [comment]
Hi @sylvain-viole 
Thank you for posting an issue and for your investigation.
In case parameter has value of non-primitive type it will throw and error when you will try to generate report.
So will handle this right from plugin to avoid confusion.

**Comment by Shelex** [comment]
Hi @sylvain-viole 
Now handling this case by trowing error, released in v2.24.0
